src/GPL-1.0.xml

```diff
@@ -0,0 +1,242 @@
+<spdx name="GNU General Public License v1.0 only" identifier="GPL-1.0" osi-approved="false">
```

Comment:
Why the “ only”?  Won't that be true of all the entries in this repository?  Before 2.0, there were explicit entries for [`GPL-1.0+`][1], etc., but now that is covered by the `+` [license expression syntax][2] (so `GPL-1.0+` is still a valid expression, but is no longer a license ID.

[1]: https://github.com/spdx/license-list/blob/788e54b566dc78358e3e1c3f9b7187ad4005114e/deprecated_GPL-1.0%2B.txt
[2]: https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60